spambot.install

```diff
@@ -96,8 +102,9 @@ function spambot_update_1000() {
 }
 
 /**
- * Implements hook_install().
+ * Implements hook_update_N(). Move last checked timestap to state system.
```

Comment:
```suggestion
 * Move last checked timestamp to state system.
```

Implementations of `hook_update_N()` don't use the "Implements..." in their docblock, because that text is used on the update page. Also fixed spelling of "timestamp".